File: solutions/python/raindrops/1/raindrops.py
def convert(number):
    message_list = []
    if number % 3 == 0:
        message_list.append("Pling")
    if number % 5 == 0:
        message_list.append("Plang")
    if number % 7 == 0:
        message_list.append("Plong")
    return "".join(message_list) or str(number)

File: solutions/julia/raindrops/1/raindrops.jl
function raindrops(number::Int)
    drops = []
    number % 3 == 0 && push!(drops, "Pling")
    number % 5 == 0 && push!(drops, "Plang")
    number % 7 == 0 && push!(drops, "Plong")
    size(drops, 1) == 0 ? repr(number) : join(drops)
end